cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release_test.go

```diff
@@ -816,16 +834,20 @@ var redis_summary_latest = &corev1.InstalledPackageSummary{
 	PkgVersionReference: &corev1.VersionReference{
 		Version: "*",
 	},
-	CurrentPkgVersion: "14.4.0",
-	CurrentAppVersion: "6.2.4",
-	PkgDisplayName:    "redis",
-	ShortDescription:  "Open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.",
+	CurrentVersion: &corev1.PackageAppVersion{
+		PkgVersion: "14.4.0",
+		AppVersion: "6.2.4",
+	},
+	PkgDisplayName:   "redis",
+	ShortDescription: "Open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.",
 	Status: &corev1.InstalledPackageStatus{
 		Ready:      true,
 		Reason:     corev1.InstalledPackageStatus_STATUS_REASON_INSTALLED,
 		UserReason: "ReconciliationSucceeded",
 	},
-	LatestPkgVersion: "14.6.1",
+	LatestVersion: &corev1.PackageAppVersion{
```

Comment:
yeah, I noticed flux CRD doesn't give me an app version, only pkg version. I guess I will have to pull that from the cached index, if/when needed